docs/advanced/command-line/files/sources.md

```diff
@@ -0,0 +1,36 @@
+# Sources
+
+The file describing how to access the sources, passed as an argument to the -s command line option
+
+## Format of the file
+We have provided an example of how the contents of the file should be structured in **sources.csv** in the Manual Books example.
+
+The csv file is expected to contain exactly the following columns:
+
+1. uri. A jdbc connection string to an external database, as described in [Adding Data Uri](../../../../guides/adding-data/local-DB/). If a uri
+is not specified here, then the source entry should be defined in column data.
+
+2. tables. Optionally, in case a uri is provided, a comma-separated list of the specific tables in the database schema to consider in the wrangling process. If none is specified, all tables will be considered.
+
+3. description. Optionally, a description of the resource.
+
+4. data. A file directory, or a delimiter-separated text file. The file path can be either absolute, or relative to the location of the Data Preparer executable. The first line in each source file is expected to contain the field names. The delimiter can optionally be defined
+in the separator column.
+
+5. extensions. A comma-separated list of file extensions. In case a directory is specified in the data column, we can optionally filter by extension the files to wrangle. E.g. csv, tsv.
+
+6. separator. Optionally, an explicit separator character to use when parsing a delimiterseparated text file defined in the data column. The delimiter can be a comma, a tab, a colon, or a pipe (, or \t or : or |). If none is specified, the separator character will be autodetected.
+
+7. recursive. In case a file directory is specified in the data column, we defined here whether the system should also scan its subdirectories for data files to import. 
+> Default is false.
```

Comment:
You can add "Note: " before each comment like this.

Reply:
I'm more of using the comments to emphasize the defaults because the bold and italics is not good enough